Media query homework/style.css

```diff
@@ -0,0 +1,59 @@
+.container {
+    display: grid;
+    grid-template-columns: 150px 500px 150px;
+    grid-template-rows: 30px 100px 30px;
+    width: 800px;
+    height: 300px;
+}
+
+.header {
+    display: flex;
+    background-color: rgb(232, 100, 100);
+    grid-column-start: 1;
+    grid-column-end: 4;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    justify-content: center;
+    align-items: center;
+}
+
+.aside1 {
+    display: flex;
+    background-color: yellow;
+    grid-row-start: 2;
+    justify-content: center;
+}
+.center {
+    display: flex;
+    background-color: rgb(99, 99, 219);
+    justify-content: center;
+    text-align: center;
+    align-items: center;
+}
+.aside2 {
+    display: flex;
+    background-color: pink;
+    grid-column-start: 3;
+    grid-column-end: 4;
+    grid-row-start: 2;
+    grid-column-end: 3;
+    justify-content: center;
+}
+
+.footer {
+    display: flex;
+    background-color: rgb(101, 185, 101);
+    grid-column-start: 1;
+    grid-column-end: 4;
+    grid-row-start: 3;
+    grid-row-end: 4;
+    justify-content: center;
+    text-align: center;
+    align-items: center;
+}
+
+@media (max-width: 720px) {
+    .container {
+      grid-template-columns: 100px;
```

Comment:
Aici ar fi trebuit sa modifici din display:grid in display: flex cu flex-direction:column;
Apoi sa vezi cum poti reordona elemenlele pentru ca sa fie pozitionate ca in tema. Asta se face cu proprietatea order: